test/integ/test_timestamp_binding.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+from datetime import datetime
+from typing import List
+
+import pytest
+
+from snowflake.connector.cursor import SnowflakeCursor
+
+from ..randomize import random_string
+
+
+def create_or_replace_table(cur: SnowflakeCursor, table_name: str, columns: List[str]):
+    sql = f"CREATE OR REPLACE TEMP TABLE {table_name} ({','.join(columns)})"
+    cur.execute(sql)
+
+
+def insert_multiple_records(
+    cur: SnowflakeCursor,
+    table_name: str,
+    ts: str,
+    row_count: int,
+    should_bind: bool,
```

Comment:
what does should_bind do?
I see in the test we always set this to True, do we want to parameterize this as well?